lib/base/RequestClient.js

```diff
@@ -41,7 +40,9 @@ RequestClient.prototype.request = function(opts) {
   var deferred = Q.defer();
   var headers = opts.headers || {};
 
-  if (!headers.Connection && !headers.connection) {
+  if (!headers.Connection && !headers.connection && opts.forever) {
+    headers.Connection = 'keep-alive';
```

Comment:
nit: `headers.Connection = opts.forever ? 'keep-alive' : 'close';`

Reply:
Hi. Apologies. Not following. NIT?

I think you are suggesting there is an alternate better approach here?

Reply:
[nit](https://stackoverflow.com/questions/27810522/what-does-nit-mean-in-hacker-speak)

Just suggesting a less-verbose replacement that's functionally equivalent.